website/content/docs/configuration/request-limiter.mdx

```diff
@@ -0,0 +1,32 @@
+---
+layout: docs
+page_title: Request Limiter - Configuration
+description: |-
+  The Request Limiter mitigates overload scenarios in Vault by adaptively
+  limiting in-flight requests based on latency measurements.
+---
+
+# `request_limiter`
+
+@include 'alerts/enterprise-only.mdx'
+
+@include 'alerts/beta.mdx'
+
+The `request_limiter` stanza allows operators to turn on the adaptive
+concurrency limiter, which is off by default. This is a reloadable config.
+
+```hcl
+request_limiter {
+  disable = false
+}
+```
+
+~> **Warning** This feature is still in Tech Preview. Turning the Request
+Limiter *on* may have negative effects on request success rates. Please test
+your workloads before turning this on in production.
+
+~> **Warning** Turning the Request Limiter *off* while Vault is experiencing
+high levels of traffic could result in immediate overload of the server. Great
+care should be taken when using this configuration to avoid adverse effects in
+optimal request servicing. It is strongly advised NOT to use this configuration
```

Comment:
DO you mean "NOT to use this configuration _during an incident_ without very good reason"?

Or do you mean specifically that disabling once it's already enabled is generally a bad idea? Surely only if it's overloaded?

As it is it's ambiguous whether you are suggesting that toughing the request limiter _at all_ (even to experiment with it) is not recommended which seems odd!

Reply:
I removed this in https://github.com/hashicorp/vault/pull/25557/commits/7a38c1ff7ee0448f14b115478f967947a06e704e